src/app/embed/share/share-modal.component.ts

```diff
@@ -5,7 +5,7 @@ import { Component, EventEmitter, Output, Input } from '@angular/core';
   styleUrls: ['share-modal.component.css'],
   template: `
     <button class="close-btn" (click)="closeModal()" (window:keydown)="handleKeypress($event)">✖</button>
-    <template [ngIf]="mode == 'share'">
+    <ng-template [ngIf]="mode == 'share'">
```

Comment:
this is an angular change, `<template/>` is too generic, so they use `<ng-template/>`